system/Commands/Utilities/Routes/AutoRouterImproved/ControllerMethodReader.php

```diff
@@ -164,7 +166,7 @@ private function getParameters(ReflectionMethod $method): array
      *
      * @return string URI path part from the folder(s) and controller
      */
-    private function getUriByClass(string $classname): string
+    private function convertClassnameToUri(string $classname): string
```

Comment:
```suggestion
    private function convertClassNameToUri(string $classname): string
```
Simple for the name consistency, we have `convertMethodNameToUri` method below.

Reply:
Done.

Reply:
Thanks!